care_scribe/tasks/scribe.py

```
@@ -87,200 +129,173 @@ def chat_message(provider=plugin_settings.SCRIBE_API_PROVIDER, role="user", text
-        tnc = plugin_settings.SCRIBE_TNC
-        tnc_hash = hash_string(tnc)
+        if not is_benchmark:
+            user_quota: ScribeQuota | None = form.requested_by.scribe_quota.filter(
+                facility=form.requested_in_facility
+            ).first()
+            facility_quota: ScribeQuota | None = form.requested_in_facility.scribe_quota.filter(user=None).first()
 
-        if user_quota.tnc_hash != tnc_hash:
-            error = "User has not accepted the latest terms and conditions."
+            if not facility_quota:
+                raise ScribeError("Facility does not have a scribe quota.")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)